src/main/steps/applicant1/confirm-your-joint-application/template.njk

```diff
@@ -1,37 +1,81 @@
-{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
```

Comment:
this whole template seems to be similar to review the application template, maybe you should reuse that.

Reply:
I changed.  I copied the review application template and changed to fit what I need for this one